plugin.rb

```diff
@@ -36,6 +36,16 @@
 
   add_to_class(:post, :has_jira_issue?) { custom_fields["jira_issue"].present? }
 
+  add_to_class(:post, :jira_issue_key=) do |key|
```

Comment:
Do you think it's worth porting this functionality to the custom fields concern? Looks pretty nice. 🙂

Reply:
I think it may not be useful all the time. For example, if someone mistakenly writes `post.create_at = Time.now` then it will save it in custom fields instead of raising an error.